src/backends/torch/torchlib.cc

```diff
@@ -1033,7 +1064,19 @@ namespace dd
         throw;
       }
     this->_stats.transform_end();
-
+    if (_dtype == torch::kFloat16)
+      {
+        this->_logger->info("performing inference in FP16");
```

Comment:
I don't believe we want to log this for every inference. I suggest we either remove the logging, or log it once at service creation / mllib init only.

Reply:
ok, done, it wasa debugging  log ;)

Reply:
![image](https://user-images.githubusercontent.com/3530657/113394481-9d66bc80-9398-11eb-9c65-ce3577501eda.png)
 ?

Reply:
rooooh !